app/src/main/res/values/strings.xml

```diff
@@ -236,14 +236,14 @@
     <string name="unsupport_magisk_msg">This version of the app does not support Magisk version lower than %1$s.\n\nThe app will behave as if no Magisk is installed, please upgrade Magisk as soon as possible.</string>
     <string name="unsupport_general_title">Abnormal State</string>
     <string name="unsupport_system_app_msg">Running this app as a system app is not supported. Please revert the app to a user app.</string>
-    <string name="unsupport_other_su_msg">An \"su\" command that does not belong to Magisk is detected. Please remove the other unsupported su.</string>
+    <string name="unsupport_other_su_msg">Please remove the app that just tried to handle a \"su\" command and handle all requests through Magisk instead.</string>
```

Comment:
```suggestion
    <string name="unsupport_other_su_msg">Please remove the competing \"su\" rooting solution on your device by uninstalling it.</string>
```

Reply:
Still needlessly wordy. "remove" it "by uninstalling it" is redundant. The one I proposed in the other thread is still more direct and succinct for a technical error message like this:

```
A \"su\" binary not from Magisk has been detected. Please remove any competing root solution and/or reinstall Magisk.
```

Reply:
Revised it with that in mind. It can't be more direct than a requested action.

Reply:
And to be clear, the original "an" before "su" wasn't wrong (entirely depending on how you pronounce "su", of course):
https://www.merriam-webster.com/words-at-play/is-it-a-or-an

Reply:
Yeah the "an" wasn't his, so we're good there.

Reply:
@Didgeridoohan Not being wrong doesn't mean OK. For as many people that happen to pronounce it differently, the chosen solution looks wrong.